node/impl/storminer.go

```diff
@@ -48,6 +48,8 @@ import (
 type StorageMinerAPI struct {
 	fx.In
 
+	Subsystems *api.MinerSubsystem `optional:"true"`
```

Comment:
```suggestion
	EnabledSubsystems *api.MinerSubsystem `optional:"true"`
```

Reply:
Is this truly optional? I would imagine it's not.

Reply:
It's not, removed the `optional` tag.